### PR TITLE
FHB-381 build warnings (part 1)

### PR DESCRIPTION
**Pull request by stevesatdfe**
Part 1 - fix warnings

Note: Temp addition of , feature/fhb-381-build-warnings build trigger - will remove at end

### CODE REVIEW

**Comment by AaronJY** [comment]
Just tweaking the title so Jira can pick it up :)

**Comment by AaronJY** [comment]
What's coming in part two, just so I know what the scope of this PR is

File: src/service/notification-api/shared/FamilyHubs.Notification.Api.Contracts/MessageDto.cs [hunk 1]
```diff
@@ -1,11 +1,15 @@
-﻿namespace FamilyHubs.Notification.Api.Contracts;
+﻿// ReSharper disable PropertyCanBeMadeInitOnly.Global
+// ReSharper disable NonReadonlyMemberInGetHashCode
+// ReSharper disable UnusedMember.Global
+// ReSharper disable UsageOfDefaultStructEquality
```
Comment (Zac-AND-Digital) [inline src/service/notification-api/shared/FamilyHubs.Notification.Api.Contracts/MessageDto.cs:4]: Are these ReSharper comments just a temporary thing that will be addressed later?

Comment (stevesatdfe) [reply src/service/notification-api/shared/FamilyHubs.Notification.Api.Contracts/MessageDto.cs:4]: There are nearly 2000 warnings overall. Some are simple like namespaces and others are more complicated. In this case its attached to an object that gets serialized so I don't want to change everything to get, init and in one property, its written to.

**Pull request closed without merging**